performance-timeline/po-observe.html

```diff
@@ -74,9 +74,9 @@
         buffered: true
       });
 
-      const timeout = setTimeout(function() {
+      const timeout = step_timeout(function() {
         assert_unreached("the buffered entries were not observed within 1 second of registration");
```

Comment:
I checked with @jgraham and this is will likely to make the test unstable in CIs. It seems better to let the test timeout.